bundles/org.openhab.binding.radiothermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -31,6 +31,7 @@
 			<channel id="today_cool_runtime" typeId="today_cool_runtime"/>
 			<channel id="yesterday_heat_runtime" typeId="yesterday_heat_runtime"/>
 			<channel id="yesterday_cool_runtime" typeId="yesterday_cool_runtime"/>
+			<channel id="message" typeId="message"/>
```

Comment:
Please add update instructions for this new channel - see https://github.com/openhab/openhab-docs/blob/088e0929d467d23014af613c1b3ced389c8ab0b9/developers/bindings/thing-xml.md#updating-thing-types

Reply:
cool, now I don't have to delete and re-create the thing to add channels!